package.json

```diff
@@ -151,6 +151,7 @@
     "proxyquire": "2.1.0",
     "ramda": "0.24.1",
     "shelljs": "0.8.3",
+    "shx": "^0.3.2",
```

Comment:
pin it, or else renovatebot will open a PR to pin it iirc

`"0.3.2"`

Reply:
nice, added `save-exact` to `.npmrc` to help with this